tasks/install-OpenBSD.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+- name: Create ports directory
+  shell: ftp -o - https://ftp.openbsd.org/pub/OpenBSD/6.0/ports.tar.gz | tar -C /usr -zxf -
+  args:
+    creates: /usr/ports/Makefile
+  register: register_ftp_o_tar_C_zxf
+
+- name: Add ssh key to known_hosts
+  known_hosts:
+    path: /etc/ssh/ssh_known_hosts
+    state: present
+    name: anoncvs.ca.openbsd.org
+    key: "{{ item }}"
+  with_file:
+    - files/anoncvs.ca.openbsd.org
+
+- name: Update ports tree
+  # XXX the git repository, unofficially published at github.com, cannot be used
+  # here because the repository does not have tags, but only HEAD. HEAD can only
+  # be built on -current. To support git repository, one must convert a tagged
+  # CVS branch to a git branch, and tag it.
+  command: cvs -qd anoncvs@anoncvs.ca.openbsd.org:/cvs get -rOPENBSD_6_0 -P ports
+  args:
+    chdir: /usr
+  when:
+    - register_ftp_o_tar_C_zxf.changed
```

Comment:
probably this part would be modified when the role supports other version control system. i am not sure how it would be. the use case at the moment is creating a working build environment with the latest ports tree.